Specs/VSLAM.txt

```diff
@@ -0,0 +1,5 @@
+Localize ourself on the field using a camera
+
+Needs relocalization
+Needs a method of denoising the recived data
```

Comment:
Noise and drift are two different things.

Reply:
Yup, denoising doesn't necessarily fix drift.